src/scenes/ProgressReports/Detail/WorkflowEventList.tsx

```diff
@@ -0,0 +1,71 @@
+import { ArrowForward } from '@mui/icons-material';
+import { Box, Card, Collapse, Divider, Stack, Typography } from '@mui/material';
+import { ProgressReportStatusLabels as StatusLabels } from '~/api/schema/enumLists';
+import { FormattedDate } from '~/components/Formatters';
+import { RichTextView } from '~/components/RichText';
+import { WorkflowEventFragment } from './WorkflowEvent.graphql';
+
+interface WorkFlowEventListProps {
+  events?: readonly WorkflowEventFragment[];
+  showNotes?: boolean;
+}
+
+export const WorkFlowEventList = ({
+  showNotes,
+  events,
+}: WorkFlowEventListProps) => {
+  if (!events) {
+    return null;
+  }
+
+  const statusEvents = [
+    {
+      ...events[0]!,
+      status: 'NotStarted' as keyof typeof StatusLabels,
+    },
+    ...events,
+  ];
+
+  return (
+    <Card sx={{ p: 4, overflowY: 'auto' }} elevation={0}>
```

Comment:
Why deviate on `elevation` here?

Reply:
I was just following what the design shows.  The design does not have elevation.
I can change it I just wasn't sure what freedoms we can take with the design.  I did notice you did not include it in your example but I just thought you were not being specific about the elevation.